spacemouse-keys/hidInterface.cpp

```diff
@@ -6,21 +6,94 @@
 
 // Array with the bitnumbers, which should be assign keys to buttons
 uint8_t bitNumber[NUMHIDKEYS] = BUTTONLIST;
+#if HIDUPDATERATE_MS > 0
+unsigned long lastHIDsentRep1 = 0; // time from millis(), when the last HID report was sent
+unsigned long lastHIDsentRep2 = 0; // time from millis(), when the last HID report was sent
```

Comment:
I think that those 2 variables introduced more complexity than needed: `HID().SendReport(1, trans, 6);` and `HID().SendReport(2, rot, 6);` can be sent together with the same frequency even if there is no movement. This will also remove the need for the jiggle logic.

The previous code was sending to much data, this is why on my version, I have a `wait_ms()` in the main loop, I submitted it to you in the case of the debug but I was testing it. This new implementation it better but don't not try to make it too complex please :-)

Reply:
Yeah, this was more complex than needed, granted. I merged those two parts together in a961ae9.

I still need those jiggling for my ubuntu + spacenavd driver issues. Is it clear enough in the config, that is a special feature not for everyone? Than I would prefer to leave it in place.

Reply:
Very nice rework ! More concise :)